website/blog/2019-01-21-jest-24-new-website-babel-7-and-typescript-support.md

```diff
@@ -0,0 +1,94 @@
+
+```js
+test.todo('invalid input should throw');
+
+test.todo('missing options should be normalized');
+```
+
+![test.todo](/img/blog/24-todo.png)
+
+## Improved Assertion Messages
+
+When tests fail, you need to make confident and correct decisions which changes are expected progress and which changes are unexpected regressions. It is especially important not to miss a few regressions hidden among much progress. Jest 24 makes reports when assertions fail more clear and concise for several matchers. Because the effort will continue in Jest 25, you might notice some temporary inconsistencies. If your tests never fail, then you won't get to see them - for the rest of us, it'll be easier to debug why something isn't working as expected. Thanks for the hard work by [@ittordepam](https://twitter.com/ittordepam)[](https://twitter.com/ittordepam) and other contributors from the community.
+
+You can see these changes across all these PRs: [7557](https://github.com/facebook/jest/pull/7557), [7621](https://github.com/facebook/jest/pull/7621), [7557](https://github.com/facebook/jest/pull/7557), [7457](https://github.com/facebook/jest/pull/7457), [7448](https://github.com/facebook/jest/pull/7448), [7241](https://github.com/facebook/jest/pull/7241), [7152](https://github.com/facebook/jest/pull/7152), [7125](https://github.com/facebook/jest/pull/7125),[7107](https://github.com/facebook/jest/pull/7107), [6961](https://github.com/facebook/jest/pull/6961).
+
+Examples:
+
+Failing assertion
+
+![failing assertion](/img/blog/24-assertion-error.png)
+
+Type mismatch
+
+![different types](/img/blog/24-different-types.png)
+
+Mock function not called
+
+![mock functions](/img/blog/24-mock-function.png)
+
+## Fixing old pain
+
+We've had a couple of really old issues fixed by this release.
+
+The first one we'd like to highlight is `console.log` statements going missing. Jest intercepts and collects all logs in order to give you a stack trace to them, as well as provide them to reporters so you can handle them however you want. However, this has led to an issue where they have simply been missing in certain edge cases. Luckily for Jest 24, [@spion](https://twitter.com/spion) has [stepped](https://github.com/facebook/jest/pull/6871) up and fixed this issue. Thank you very much!
+
+The second one is an issue where Jest runs out of memory if the difference in serialization of expected and received value has a huge number of insertion changes (either unexpected because of mistake in test or defect in serializer or expected because of temporary failures during test-driven development). [@ittordepam](https://twitter.com/ittordepam) has [replaced](https://github.com/facebook/jest/pull/6961) the previous diffing algorithm with `diff-sequences` package, which should fix this issue because it uses the theoretical minimum amount of memory. It also opens up for word-diffing in the future.
+
+## Other highlights
```

Comment:
There are a few breaking changes buried in here we should move to a `## Major Breaking` section or otherwise call out louder